common/app/layout/Front.scala

```diff
@@ -476,8 +489,11 @@ object Front extends implicits.Collections {
         (Set.empty[TrailUrl], initialContext, emptyDedupedResultWithPath)
       ) { case ((seenTrails, context, dedupedFrontResult), (pressedCollection, index)) =>
         val omitMPU = pressedPage.metadata.omitMPUsFromContainers(edition)
+        val isNetworkFront = pressedPage.metadata.isNetworkFront
         val container = Container.fromPressedCollection(pressedCollection, omitMPU)
-        val (newSeen, newItems, (usedAndDeduped, usedButNotDeduped)) = deduplicate(seenTrails, container, pressedCollection.curatedPlusBackfillDeduplicated)
+        val isEditorialFront = ConfigAgent.getPriorityForPath(pressedPage.id).getOrElse("editorial") == "editorial"
```

Comment:
Ah yes, maybe there are a few now laying around that have existed in the past when we didn't have `priority`!